R/calcIVarCost.R

```diff
@@ -1,7 +1,7 @@
 #' calcIVarCost
 #'
 #' Use data from EU Reference Scenario to derive OPENPROM input parameter iVarCost
-#' This dataset includes variable non fuel cost per plant type, in EUR/MWh.
+#' This dataset includes variable non fuel cost per plant type, in EUR/KW.
```

Comment:
I think that the units here were right

Reply:
@giannou There's a discrepancy between the units used in Reference Scenario 2020 (EUR/MWh), OPEN-PROM code (EUR/KW) and the old MENA-EDS dataset (mEuro/KWh), with the numeric values being otherwise identical. Which unit should we use?

Reply:
Variable non fuel costs should be measured per energy units, not power. So the Reference Scenario units should be the correct ones.